### ISSUE TITLE
r6sapi.Operator.statistic always returns 0

### ISSUE THREAD

**Issue by Fozzieponken** [billy-yoyo/RainbowSixSiege-Python-API#8]
I've been using this API to a discord hobby project and everything have been working awesome (so thanks for that!) with the exception that every time I try to find the 'statistic'-value for an operator it always returns 0. All other attributes return the expected value.

**Comment by billy-yoyo** [comment]
Are you on the latest version? I can only reproduce this with the newest operators (vigil, zofia, dokk) and that's because Ubi hasn't updated their API to include unique statistics for them yet.

Sorry for the slow reply!

**Comment by Fozzieponken** [comment]
I'm on the latest version. Just to be on the safe side I ran --upgrade on your package just now, but it did nothing. After I loaded the operator I tried to print all the parameters on the object and in the case of statistics I simply get this, for example with Valkyrie:

print(p_operator.kills)
print(p_operator.statistic_name)
print(p_operator.statistic)
>> 353
>> Cameras Deployed
>> 0

**Comment by billy-yoyo** [comment]
I couldn't reproduce, I did the following:
upgraded r6sapi `pip install r6sapi -U` in the command line
then I ran the following code:
```py
import r6sapi
import asyncio

@asyncio.coroutine
def run():
    auth = r6sapi.Auth("email", "password")
    player = yield from auth.get_player(name="billy_yoyo", platform=r6sapi.Platforms.UPLAY)
    op = yield from player.get_operator("valkyrie")
    print("%s: %s" % (op.statistic_name, op.statistic))

asyncio.get_event_loop().run_until_complete(run())
```
and got the following output:
```
Cameras Deployed: 240
```

Could you give me the code you're using to load the operators? 
Also could I ask where you're from, I guess it's possible this is a problem with the CDN, which I'd have to look in to if that was the case.

**Comment by Fozzieponken** [comment]
Hmm. My friend commented on the possibility of a CDN problem as well. I live in Sweden, so continent Europe. Anyway, the code I am running currently looks like this: 

```
  await acc.load_all_operators()
  p_operator = acc.operators.get(second_paramter)
  if p_operator == None:
        await auth.session.close()
        await client.send_message(message.channel, 'Some reply')
        return
 result_list.append((p_operator.statistic_name, str(p_operator.statistic)))
 print(p_operator.statistic_name)
 print(p_operator.statistic)
```

Which gives the output from my earlier reply.

**Comment by billy-yoyo** [comment]
ah, found the issue! `load_all_operators()` wasn't saving the statistic name properly, but `load_operator` was, which was why you were getting the problem but I wasn't.

I just committed a fix, I'll update the PyPI package ASAP as well.

**Comment by billy-yoyo** [comment]
fixed in v0.5.1 :)
